lidar_cluster_topic_name: /perception/lidar_cluster

line_path_topic_name: /planning/global_path

node_rate: 50   # [Herz]

path_length: 80 # [m]
